src/include/binder/copy/bound_copy_from.h

```diff
@@ -33,6 +33,8 @@ struct KUZU_API BoundCopyFromInfo {
     std::vector<common::ColumnEvaluateType> columnEvaluateTypes;
     std::unique_ptr<ExtraBoundCopyFromInfo> extraInfo;
 
+    explicit BoundCopyFromInfo(catalog::TableCatalogEntry* tableEntry)
```

Comment:
Where are we using this constructor?

Reply:
It's used inside `getPhysicalPlan`  within `create_hnsw_index.cpp`.